.automation/add-to-project-when-labeled

```diff
@@ -38,15 +38,26 @@ except KeyError:
     content_id = event["pull_request"]["id"]
 
 # Fetch the project we want to add to or remove from
-project = [p for p in repo.get_projects() if p.name == LABEL_PROJECTS[label_name]][1]
+try:
+    project = [p for p in repo.get_projects() if p.name == LABEL_PROJECTS[label_name]][0]
+except KeyError:
+    print("Label is not actionable.")
+    sys.exit(0)
 
 # If the label was added, we want to add to the project. If removed, we want to
 # remove from the project.
 if action == "labeled":
     project_column = project.get_columns()[1]
-    project_column.create_card(content_id=content_id, content_type=content_type)
+    try:
+        project_column.create_card(content_id=content_id, content_type=content_type)
+    except github.GithubException as e:
+        if e.status == 422:
```

Comment:
Don't use a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)).

Reply:
Noted. Latest push fixes this.